src/testing.rs

```diff
@@ -543,7 +543,7 @@ impl StorageRunner {
     }
 
     fn new(desc: StoreDescription) -> Result<Self> {
-        Self::new_with_path(desc, TempDir::new("atomicstore_test").unwrap())
+        Self::new_with_path(desc, TempDir::new().unwrap())
```

Comment:
Not that it really matters, but the direct equivalent of the old code is apparently 
`use tempfile::Builder;`

`Self::new_with_path(desc, Builder::new().prefix("atomicstore_test").tempdir().unwrap())`

... but what you have is really perfectly fine.